### DISCUSSION TITLE
Relative paths of images

### DISCUSSION THREAD

**Discussion by sorsted** [Witiko/markdown#385] [Q&A]
According to how markdown is usually interpreted, file paths are understood relatively to the file at hand. So if I have the following document, called `chap1.md`, then the file `tug.png` is supposed to be in the same folder as `chap1.md`:

```markdown
Hello world:

![tug](tug.png)
```

But let’s suppose that `chap1.md` and `tug.png` are both placed in the folder `content`, and that the parent folder contains the following LaTeX document. Then LaTeX will look for `tug.png` in the parent folder rather than the folder of the markdown document, hence it will be unable to find it and will issue an error instead.

```latex
\documentclass{article}

\usepackage{markdown}

\begin{document}

\markdownInput{content/chap1.md}

\end{document}
```

**Comment by Witiko** [comment]
You can use the the `\graphicspath` command to specify where LaTeX should look for images:

``` tex
\usepackage{graphicx}
\graphicspath{{./}{content/}}
```

With this command, images are first searched in the parent folder and then in the folder `content/`.
